junorunner/runner.py

```diff
@@ -108,8 +100,8 @@ def _get_parallel_suite(self, suite):
 
     def build_suite(self, test_labels=None, extra_tests=None, **kwargs):
         extra_tests = extra_tests or []
-        test_labels, methods = self._parse_labels_and_methods(test_labels)
 
+        methods = self.methods.split(',') if self.methods else []
```

Comment:
I assume the colon functionality is not necessary? Couldn't find what it's even used for.

Reply:
it is used in next row...